apps/dashboard/app/views/batch_connect/sessions/card/_session_time.html.erb

```diff
@@ -0,0 +1,25 @@
+<% time_limit = session.info.wallclock_limit %>
+<% time_used  = session.info.wallclock_time %>
+<p>
+  <% if session.starting? || session.running? %>
+    <% if time_limit && time_used %>
+      <strong> 
+        <%= t('dashboard.batch_connect_sessions_stats_time_remaining') %> 
+      </strong>
+      <%= distance_of_time_in_words(time_limit - time_used, 0, false, :only => [:minutes, :hours], :accumulate_on => :hours) %>
```

Comment:
Given that there's so much logic in this partial - I'd say let's pull all that out to a helper method.  I'm thinking the return value to be 2 item array, like so:

```ruby
def session_time_used
  # all that if logic
  [label, value]
end
```
So this partial becomes as simple as:
```erb
<%- label, value = session_time_used(session) -%>
<p>
  <strong><%= label %></strong>: <%= value %>
</p>
```

Reply:
would i put this helper method within the partial file?